draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -1429,6 +1429,53 @@ A DCCP implementation maintains the maximum packet size (MPS) during operation o
 
 For compatibility reasons, an MP-DCCP implementation SHOULD always announce the minimum MPS across all paths.
 
+## Path usage strategies
+
+MP-DCCP can be configured to realise one of several strategies for path usage, via selecting one DCCP subflow of the multiple DCCP subflows within a MP-DCCP connection for data transmission. This can be a dynamic process further facilitated by the means of DCCP and MP-DCCP defined options such as path preference using MP-PRIO, adding or removing DCCP subflows using MP_REMOVEADDR, MP_ADDADDR or DCCP-Close/DCCP-Reset and also path metrics such as packet-loss-rate, CWND or RTT provided by the Congestion Control Algorithm.
+Selecting an appropriate method can allow MP-DCCP to realise different path utilization strategies that make MP-DCCP suitable for end-to-end implementation over the Internet or in controlled environments such as Hybrid Access or 5G ATSSS.
+
+### Path mobility {#path_mobility}
+
+The path mobility strategy provides the use of a single path with a seamless handover function to continue the connection when the currently used path is deemed unsuitable for service delivery.
+Some of the DCCP subflows of a MP-DCCP connection might become inactive due to either the occurrence of certain error conditions (e.g., DCCP timeout, packet loss threshold, RTT threshold, closed/removed) or adjustments from the MP-DCCP user.
+When there is outbound data to send and the primary path becomes inactive (e.g., due to failures) or de-prioritized, the MP-DCCP endpoint SHOULD try to send the data through an alternate path with a different source or destination address (depending on the point of failure), if one exists. This process SHOULD respect the path prio configured by MP_PRIO or if not available pick the most divergent source-destination pair from the original used source-destination pair.
+Note: Rules for picking the most appropriate source-destination pair are an implementation decision and are not specified within this document.
+Path mobility is supported in the current Linux reference implementation {{website}}.
+
+### Concurrent path usage
+Different to a path mobility strategy, the selection between MP-DCCP
+subflows is here a per-packet decision that is a part of the multipath
+scheduling process. This method would allow multiple subflows to be
+simultaneously used to aggregate the path resources to obtain higher
+connection throughput.
+
+In this scenario, the selection of congestion control, per-packet scheduling
+and potential re-ordering method determines a concurrent path utilization
+strategy and result in a particular transport characteristic.
+
+The concurrent path usage method allows a scheduling design to be used to
```

Comment:
-Being extra caerful, I'd prefer to avoid the word "allow" here, simply to avoid that specific sentence being cited back me in later years. Could we simply write:
/The concurrent path usage method allows a scheduling design to be used to realise various designs of multipath usage. Scheduling could/
A concurrent path usage method uses a scheduling design that could/

Reply:
Resolved in https://github.com/markusa/ietf-multipath-dccp/pull/190/commits/970480e3835bded1554b5e7f5a95f6a6660084b1